deploy/cluster-monitoring-config-uwm/cluster-monitoring-config.yaml

```diff
@@ -12,6 +12,7 @@ data:
     # so that killswitch instructions point to the right file!
       remoteWrite:
         - url: http://token-refresher.openshift-monitoring.svc.cluster.local
+          max_samples_per_send: 100
```

Comment:
@RiRa12621 where is this requirement coming from?

Reply:
Was a recommendation from the observatorium team

Reply:
The configuration document from the observatorium team recommended this change.